deepchecks/core/check_result.py

```diff
@@ -58,7 +56,49 @@
 TDisplayItem = Union[str, pd.DataFrame, Styler, BaseFigure, TDisplayCallable]
 
 
-class CheckResult:
+class CheckOutput:
```

Comment:
maybe let's rename it to the `BaseCheckResult`?

Reply:
sure